pkg/infrastructure/kogito_service.go

```diff
@@ -18,6 +18,15 @@ import (
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"strconv"
+)
+
+const (
+	// HTTPPortEnvVar ... Env variable to define port on which service will listen internally
+	HTTPPortEnvVar = "HTTP_PORT"
```

Comment:
Are you going to configure the images to accept this environment?
notice that data-index images uses the KOGITO_DATA_INDEX_HTTP_PORT env, and for management console there is no env exposed so far.

Reply:
if this env would be used to set the port, we'll need to modify the management console and data-index images as well.

Reply:
nevermind, I didn't see the other PR, sry lol